.gitignore

```diff
@@ -1,6 +1,9 @@
-src/config.js
+data/config.json
+data/config.js
 node_modules/
+run.bat
```

Comment:
Why are you ignoring the file that runs the bot?

Reply:
Agreed, it shoudn't be ignored